extra/ofdpa-gitlab.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
   <remove-project name="bisdn/meta-ofdpa.git" />
-  <project name="yocto-meta-layers/meta-ofdpa.git" path="poky/meta-ofdpa" remote="gitlab" revision="master"/>
+  <project dest-branch="master" name="yocto-meta-layers/meta-ofdpa.git" path="poky/meta-ofdpa" remote="gitlab" revision="480808117baf729ec942352a2aeec6c2aa064171" upstream="master"/>
```

Comment:
```suggestion
  <project name="yocto-meta-layers/meta-ofdpa.git" path="poky/meta-ofdpa" remote="gitlab" revision="480808117baf729ec942352a2aeec6c2aa064171" upstream="master" dest-branch="master"/>
```

Reply:
To align the attribute order with the default.xml

Reply:
done